helm/sematic-server/values.yaml

```diff
@@ -83,6 +83,16 @@ deployment:
     periodSeconds: 5
     successThreshold: 1
     timeoutSeconds: 1
+  startup_probe:
```

Comment:
i would set the initial delay seconds on this to be very short, like 1 sec.  period should also be short, 1 or 2 secs.  failure theshold should be set much higher, like 90 (if period is 1 second).  that way we detect startup quickly, and have room for large variance in startup delays.  these checks are cheap.